src/test/java/dev/ai4j/openai4j/completion/CompletionTest.java

```diff
@@ -41,4 +48,171 @@ void testCustomizableApi() {
 
         assertThat(response.text()).containsIgnoringCase("hello world");
     }
+
+
+    @Test
+    void shouldPreserveStreamParameter() {
```

Comment:
How does this test test anything? First of all, it is testing completion, but the changes were done in chat completion